backend/api/game.go

```diff
@@ -29,14 +53,54 @@ func getAllGames(c echo.Context) error {
 }
 
 func newGameHandler(c echo.Context) error {
-	// TODO #5 We should probably actually create a game here
-	// Demo, a change so git picks it up
+	// create new game model
+	// TODO need a security layer in between the form and our new game struct
+	newGame := models.Game{
+		Name:         c.FormValue(NAME),
+		Author:       c.FormValue(AUTHOR),
+		CreationDate: time.Now().Format("MM/DD/YYYY"),
+		Version:      c.FormValue(VERSION),
+	}
+
+	// Add new game to database
+	_db, getDbErr := db.NewDatabaseFromEnv()
+	id, err := _db.AddGame(newGame)
+
+	// error handling
+	if getDbErr != nil {
+		return c.JSON(http.StatusInternalServerError, "Database connection error")
+	}
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, "Database add game error")
+	}
+
+	// register new route with ID
+	registerRoute(route{method: http.MethodGet, path: fmt.Sprintf("/info/%s", id), handler: gameInfoHandler})
```

Comment:
Is it necessary to register a new route for each id? Couldnt we just create an info/:id route to dynamically handle requests? Might make the logic for the gameInfo handler cleaner too. 

If there is a limitation that prevents this, no worries.

Reply:
I'm not sure to be honest. I have very little web experience. I can easily revert the game info routes, and let someone with more web experience handle it.

Reply:
https://github.com/jak103/uno/blob/81104b4d3d2e7fcf1fc4820de5dfff661dba0c25/server/routeHandlers.go#L22

https://github.com/jebidiah252/usu-gdsf/blob/deae7bae0de1f741f4ba3f00ad7fa18b376b06fb/backend/api/game.go#L53

There are a couple examples of registering a route while using `game/:id` , which I think would be a better option here.

Reply:
I dont think it would be much work on top of what you have, just add the game/:id route with the other couple routes, and then update your game info handler to take the ID from context and use it the same way you already are.

Reply:
Ah, I see it just needs the ":id", and grabbing it in the handler. I can fix that.

- [ ] Update game info routes and handler